HintList.py

```diff
@@ -520,51 +520,51 @@ def tokens_required_by_settings(world):
-    'Spirit Temple MQ Symphony Room':                              ("#beyond the symphony room# in the Spirit Temple a chest and a spider hold...^", None, 'dual'),
+    'Spirit Temple Colossus Hands':                                ("upon the #Colossus's hands# lies...^", None, 'dual'),
+    'Spirit Temple Child Lower':                                   ("between the #crawl spaces in the Spirit Temple# chests contain...^", None, 'dual'),
+    'Spirit Temple Child Top':                                     ("on the path to the #right hand of the Spirit Temple# a chest and a spider hold...^", None, 'dual'),
+    'Spirit Temple Adult Lower':                                   ("past a #silver block in the Spirit Temple# boulders and a melody conceal...^", None, 'dual'),
+    'Spirit Temple MQ Child Top':                                  ("on the path to the #right hand of the Spirit Temple# a chest and a spider hold respectively...^", None, 'dual'),
+    'Spirit Temple MQ Symphony Room':                              ("#the symphony room# in the Spirit Temple protects...^", None, 'dual'),
     'Spirit Temple MQ Throne Room GS':                             ("in the #nine thrones room# of the Spirit Temple spiders hold...^", None, 'dual'),
-    'Shadow Temple Invisible Blades Chests':                       ("#invisible spinning blades# in the Shadow Temple guard...^", None, 'dual'),
-    'Shadow Temple Single Pot Room':                               ("inside a room containing #a single skull-shaped pot# are...^", None, 'dual'),
-    'Shadow Temple Spike Walls Room':                              ("behind #flammable walls# in the Shadow Temple chests contain...^", None, 'dual'),
+    'Shadow Temple Invisible Blades Chests':                       ("an #invisible spinning blade# in the Shadow Temple guards...^", None, 'dual'),
+    'Shadow Temple Single Pot Room':                               ("a room containing a #large pot in Shadow Temple# holds...^", None, 'dual'),
+    'Shadow Temple Spike Walls Room':                              ("behind #wooden walls# in the Shadow Temple hides...^", None, 'dual'),
     'Shadow Temple MQ Upper Checks':                               ("#before the Truth Spinner gap# in the Shadow Temple locked chests contain...^", None, 'dual'),
-    'Shadow Temple MQ Invisible Blades Chests':                    ("#invisible spinning blades# in the Shadow Temple guard...^", None, 'dual'),
-    'Shadow Temple MQ Spike Walls Room':                           ("behind #flammable walls# in the Shadow Temple chests contain...^", None, 'dual'),
+    'Shadow Temple MQ Invisible Blades Chests':                    ("an #invisible spinning blade# in the Shadow Temple guards...^", None, 'dual'),
+    'Shadow Temple MQ Spike Walls Room':                           ("behind #wooden walls# in the Shadow Temple chests contain...^", None, 'dual'),
```

Comment:
Needs to be consistent with vanilla.
```suggestion
    'Shadow Temple MQ Spike Walls Room':                           ("#wooden walls# in the Shadow Temple hide...^", None, 'dual'),
```